Apps/Admin/Client/Pages/DashboardPage.razor

```diff
@@ -16,58 +16,100 @@
 <MudGrid>
     <MudItem xs="12" lg="12">
         <MudPaper Class="my-2 pa-4 d-flex justify-end" Elevation="0">
-            <HgButton EndIcon="@Icons.Material.Filled.Refresh" @onclick="ReloadDispatchActions" data-testid="refresh-btn">Refresh</HgButton>
+            <HgButton Loading="@(RegisteredUsersLoading || LoggedInUsersLoading || DependentsLoading || RecurringUsersLoading)" EndIcon="@Icons.Material.Filled.Refresh" @onclick="ReloadDispatchActions" data-testid="refresh-btn">Refresh</HgButton>
         </MudPaper>
     </MudItem>
 </MudGrid>
 
-<MudGrid>
-    <MudItem xs="6" lg="6">
-        <MudPaper Class="my-2 pa-4" Outlined="true">
-            <MudText Align="Align.Center" Typo="Typo.subtitle1"><strong>Registered Users</strong></MudText>
-            <MudText Align="Align.Center" Typo="Typo.subtitle1" data-testid="total-registered-users"><strong>@TotalRegisteredUsers</strong></MudText>
-        </MudPaper>
-    </MudItem>
-    <MudItem xs="6" lg="6">
-        <MudPaper Class="my-2 pa-4" Outlined="true">
-            <MudText Align="Align.Center" Typo="Typo.subtitle1"><strong>Dependents</strong></MudText>
-            <MudText Align="Align.Center" Typo="Typo.subtitle1" data-testid="total-dependents"><strong>@TotalDependents</strong></MudText>
-        </MudPaper>
-    </MudItem>
-</MudGrid>
+@if (RegisteredUsersLoading || LoggedInUsersLoading || DependentsLoading || RecurringUsersLoading)
```

Comment:
Skeletons should only be displayed for calls that are outstanding. As each call completes, its skeleton should be replaced with the populated component/data.

Reply:
Is that how they actually want it to work?  I thought it might be a little busy if there are sections changing all over the place rather than all at once.

Reply:
I thought that was how the old admin site handled it, but it looks like it does it for recurring users and ratings but not registered users and dependents.  I would ask @ninosamson if the current implementation is suitable.

Reply:
Are we able to show them as they complete @rayking29 ? Logged in call just takes so long.

Reply:
Yes I can break it down for those 4 different loads.  The one I'm not sure about is the Rating Summary?  It is a separate component so I wasn't sure how to handle that.

Reply:
Also the date range selector and unique days parts, when do they show?

Reply:
The inputs appear right away on the old admin site.

Reply:
How about if the skeletons appearance doesn't include the logged in users, since that is only shown in the Daily Data table?  So the table and the refresh button will show as Loading until all the loads are done but the other tiles will show up without waiting for the logged in users.

Reply:
I'm still not sure that the inputs should become skeletons while data is loading, but otherwise the changes look good.